sdcm/utils/adaptive_timeouts/__init__.py

```diff
@@ -156,7 +156,7 @@ def _monitor_timeouts(self):
 
 @contextmanager
 def adaptive_timeout(operation: Operations, node: "BaseNode",  # noqa: PLR0914, F821
-                     stats_storage: AdaptiveTimeoutStore = ESAdaptiveTimeoutStore(), **kwargs):
+                     stats_storage: AdaptiveTimeoutStore = ArgusAdaptiveTimeoutStore(), **kwargs):
```

Comment:
is the mutable default here intentionally and if so, shouldnt we make it something like a class variable? I do not like relying on mutable defaults in any case

Reply:
it's not really mutable in that sense.
it's just one more way of creating a singleton

the downside of it that it's get created on loading the module (it bite us once)

Anyhow, not gonna change anything drastically in this PR